sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseDataDto.java

```diff
@@ -588,6 +588,7 @@ public class CaseDataDto extends SormasToSormasShareableDto {
 	private Map<String, String> externalData;
 	private boolean deleted;
 	private DeletionReason deletionReason;
+	@Size(max = FieldConstraints.CHARACTER_LIMIT_TEXT, message = Validations.textTooLong)
```

Comment:
this needs to be added to all other entities

Reply:
changed